dashboard-client/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "private": true,
+  "name": "dashboard-client",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "build": "preact build",
+    "serve": "sirv build --port 8080 --cors --single",
+    "dev": "preact watch",
+    "lint": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
```

Comment:
please run :)

(should also move [this](https://github.com/timothy-b/IOT-SEC/blob/b7e13cf4da2a595343222f5267148275fa13d8ff/package.json#L54) to a root .eslintrc)

Reply:
I am not sure I am understanding.

The lint rules between the service and the client will be different. So they should maintain their own eslint configs. Preact and react need specific plugins to manage things like importing their create element functions before things are transpiled. For example...

React:
```ts
import React from 'react';
```
Preact:
```ts
import { h } from 'preact';
```

without plugins, these will generate lint errors because they appear to be unused before transpilation.

Reply:
Ah, ok 👍